website/docs/reference/integrations/slack-app.md

```diff
@@ -0,0 +1,64 @@
+---
+id: slack-app
+title: Slack App
+---
+
+:::info Availability
+
+The Slack App integration was introduced in **Unleash 5.5**.
+
+:::
+
+The Slack App integration posts messages to the selected channels in your Slack workspace.
```

Comment:
Minor suggestion. Could we maybe also elaborate a bit? Don't know if this is correct, but something like that?

```suggestion
The Slack App integration posts messages to a specified set of channels in your Slack workspace. The channels can be public or private, and can be specified on a per-flag basis.
```